tests/CrewTest.elm

```diff
@@ -29,5 +29,8 @@ crewTest =
             \() ->
                 [ Slardar, Tidehunter ]
                     |> Model.Crew.suggest
-                    |> Expect.equal (Just Tusk)
+                    |> Expect.all
+                        [ Expect.notEqual (Just Slardar)
+                        , Expect.notEqual (Just Tidehunter)
+                        ]
```

Comment:
This test is now passing (in all the runs I've seen, at least), but maybe you'd like to take another look at it b/c it was returning different results every run. I changed the assertion in line with what the test description is going for, but the nondeterminism may point us to a bug or opportunity in the suggestion logic

Reply:
good enough for me. I think it was always `Tusk` because he was last alphabetically and we had a small number of heroes 🤷